.github/workflows/deploy.yaml

```
@@ -0,0 +1,15 @@
+name: Build & Deploy
+on: push
```

Kan du legge inn dette, som trengs for å få dependabot automerge til å funke
```suggestion
on:
  push:
  workflow_dispatch:
```